app/scripts/controllers/pod.js

```diff
@@ -87,6 +87,49 @@ angular.module('openshiftConsole')
       }
     };
 
+    /**
+     *  Will set the newTerm's isActive/isUsed to true, while deactivating the old's
+     */
+    $scope.onTerminalSelectChange = function(newTerm) {
+      // Deactivate all terminals
+      _.each($scope.containerTerminals, function(term) {
+        term.isActive = false;
+      });
+
```

Comment:
quick reminder, what was it that caused `newTerm` to actually be `newTerm.value` here (and can we squish `.value`)?